code/Player/Controllers/HumanController.cs

```diff
@@ -196,6 +196,8 @@ private void AccelerateGroundMovement()
 
         private void Jump()
         {
+            if (GroundEntity == ((MainPlayer)Pawn).Dragger.HeldEntity) return;
```

Comment:
nitpick: you inline the returns here and before you didn't

Reply:
Yep that's a mistake